compiler/ballerina-parser/src/test/resources/misc/typed-binding-patterns/typed_binding_patterns_source_27.bal

```diff
@@ -0,0 +1,6 @@
+public function testListBindingPatternWithInvalidMember() {
+    Arr [x, jwt:Payload] = [];
+    Arr [jwt:Header, y] = [];
+    Arr2 [jwt:Header, z, jwt:Payload] = [];
+    Arr2 [jwt:Header, jwt:Payload, jwt:Msg] = [];
```

Comment:
Using `jwt:*` is misleading because there are not binding patterns, they are types actually. Therefore shall we use some general filed-binding-patterns like `a:b` instead of `jwt:Header` and `jwt:Payload`?

Reply:
+1